app/api/probe_dock/metrics_api.rb

```diff
@@ -228,6 +245,77 @@ def count_new_tests_by_interval(start_date, interval)
         result
       end
 
+      get :testsByCategories do
+        authorize! :organization, :data
+
+        # Set filters by
+        filter_by_projects = params[:projectIds].present? && params[:projectIds].kind_of?(Array)
+        filter_by_users = params[:userIds].present? && params[:userIds].kind_of?(Array)
+
+        # Try to retrieve the project version
+        project_version = project_version()
+
+        project_versions = if !project_version.nil?
+          [ project_version.id ]
+        else
+          # Prepare a statement (postgresql) to retrieve the latest version by project
```

Comment:
Be careful with terminology. PostgreSQL has prepared statements (similar to MySQL's as far as I know), but we're not preparing a statement here. We're just dynamically building a query that will be executed later. It is not optimized like a prepared statement would be.